six/modules/c++/six.sidd/include/six/sidd/Annotations.h

```diff
@@ -35,6 +35,18 @@ struct Annotation
     std::string identifier;
     mem::ScopedCopyablePtr<SFAReferenceSystem> spatialReferenceSystem;
     std::vector<mem::ScopedCloneablePtr<SFAGeometry> > objects;
+
+    bool operator==(const Annotation& rhs) const

```

Comment:
I was looking at an old copy of `Annotations` that I had... you're right - it's all good now... we must have updated it sometime in the last month or two as part of another update.